src/management/endpoint.c

```diff
@@ -157,6 +161,25 @@ static int endpoint_create_process_data(struct controller_api *endpoint_create)
 			 endpoint_create_api->req.endpoint_name,
 			 endpoint_name_len);
 
+	rc = cfs_fs_lookup(&endpoint_name, &fs);
+        if (rc != 0) {
+                log_err("Can't create endpoint for non existent fs");
+                rc = -ENOENT;
+		request_set_errcode(request, rc);
+		endpoint_create_send_response(endpoint_create, NULL);
+                goto error;
+        }
+ 
+        /* get filesyetm ID */
+        cfs_fs_get_id(fs, &fs_id);
+
+        snprintf(str, sizeof(str), "192.%d", fs_id);
```

Comment:
Question: Will we always use 192.xxx as fs id? If yes, can we make the prefix 192 as some #define or const?